src/members/ui/ModeInfoBox.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+
+import Text from '@/common-ui/Text';
+import Icon from '@/common-ui/assets/Icon';
+import RoundedBox from '@/members/ui/RoundedBox';
+import { ColorType, theme } from '@/styles/theme';
+import getRem from '@/utils/getRem';
+
+const ModeInfoBox = ({
+  remainingDays,
+}: {
+  remainingDays: number;
+  daysPassedSinceCurrentMode: number;
+  numberOfStaleBookmarks: number;
+}) => {
+  const [isHardMode, setIsHardMode] = useState(false);
+
+  return (
+    <RoundedBox
+      backgroundColor={isHardMode ? 'lightPrimary' : 'white'}
+      style={{ position: 'relative', marginTop: getRem(21) }}
+      border={{ borderColor: 'lightPrimary' }}
+    >
+      <ModeText isHardMode={isHardMode} remainingDays={remainingDays} />
+      <ModeButton>
+        <CircleButton
+          backgroundColor={isHardMode ? 'white' : 'lightPrimary'}
+          onClick={() => setIsHardMode(!isHardMode)}
+        >
+          <Icon
+            name={isHardMode ? 'refresh-green' : 'refresh-white'}
+            size={'l'}
+          />
+        </CircleButton>
+      </ModeButton>
+      <SettingsButton>
+        <Icon
+          name={isHardMode ? 'setting-white' : 'setting-green'}
+          size={'l'}
+        />
+      </SettingsButton>
+    </RoundedBox>
+  );
+};
+
+export default ModeInfoBox;
+
+const ModeButton = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
+  top: ${getRem(23.5)};
+  right: ${getRem(20)};
+`;
+
+const SettingsButton = styled.div`
+  position: absolute;
+  bottom: ${getRem(8)};
+  right: ${getRem(10)};
+`;
+
+const CircleButton = ({
+  children,
+  backgroundColor = 'lightPrimary',
+  onClick,
+}: {
+  children: React.ReactNode;
+  backgroundColor?: ColorType;
+  onClick?: () => void;
+}) => {
+  return (
+    <StyledCircleButton
+      style={{
+        backgroundColor: theme.colors[backgroundColor],
+        cursor: onClick ? 'pointer' : 'default',
+      }}
+      onClick={
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onClick ? onClick : () => {}
+      }
```

Comment:
요거 그냥 인라인 해줘도 될것같습니당?

```suggestion
      onClick={onClick}
```